client/src/utils/prettifyStatus.js

```diff
@@ -3,16 +3,40 @@ import InfoIcon from '@material-ui/icons/Info';
 import { ComponentTooltip } from '../components/generic/ComponentTooltip';
 import { Button } from '../components/generic';
 
-export const prettifyStatus = (status, id, tabValue, handleEngage, handleAcknowledge) => {
-  let firstStatus = status[0];
+/**
+ * Returns participant stats message based on the first status provided
+ *
+ * @param {boolean} isMoH - is this view displayed for MoH user
+ * @param {string} status - first status defined in row.status
+ * @returns {string} specific status message based on value / capitalized status
+ */
+const getParticipantStatus = (isMoH, status) => {
+  if (status === 'rejected') return 'Archived';
```

Comment:
`rejected` status is not associated with MoH, consider it `Available` here.

Reply:
This is not for MoH. This function returns status for all users. MoH specific logic is captured under `if (MoH)` clause

Reply:
perfect